docs/source/user-guide/tutorials/build-pkgs.rst

```diff
@@ -99,7 +98,7 @@ Editing the meta.yaml file
        license:
        license_file:
 
-#. Save the file in the same ``pyinstrument``
+#. Save the file in the same ``click``
    directory as ``meta.yaml``. It should match :download:`this
```

Comment:
This :download: directive isn't linking to anything and it just shows a download icon. What should be on this line?

Reply:
It looks like it used to be a meta.yaml file that likely lived alongside this docs file.  If this docs file was moved in the restructure and that meta.yaml was moved or removed, it won't find it.